Cargo.toml

```
@@ -170,6 +170,7 @@ seq-macro           = "0.3.5"
 serde               = "1.0.206"
 serde_json          = "1.0.124"
 serde-wasm-bindgen  = "0.6.5"
+simdutf8            = "0.1.4"
```

Need to enable the `aarch64_neon` feature to get SIMD acceleration on ARM chipsets (e.g. Apple M series).

```suggestion
simdutf8            = { version = "0.1.4", features = ["aarch64_neon"] }
```

---

> ARM64
The SIMD implementation is only available on Rust Nightly and Rust 1.59 or later. On Rust Nightly it is now turned on automatically. To get the SIMD implementation with Rust 1.59 and 1.60 the crate feature aarch64_neon needs to be enabled. For Rust Nightly this will no longer be required (but does not hurt either). It is expected that the SIMD implementation will be enabled automatically starting with Rust 1.61.

Perhaps I misunderstood the meaning, I thought it would automatically activate.

---

Yeah, it's quite confusing. I wasn't sure either, but then I noticed that the crate has not been updated for a long time, so I assume 1.60 was latest Rust version at the time that was written. So I think that feature does need to be manually enabled (and probably won't hurt to enable it even if it doesn't need to be).